streams/src/main/java/org/apache/kafka/streams/kstream/TimeWindowedKStream.java

```diff
@@ -67,7 +67,8 @@
      * {@link StreamsConfig#CACHE_MAX_BYTES_BUFFERING_CONFIG cache size}, and
      * {@link StreamsConfig#COMMIT_INTERVAL_MS_CONFIG commit intervall}.
      * <p>
-     * For failure and recovery the store will be backed by an internal changelog topic that will be created in Kafka.
+     * For failure and recovery the store (which always will be of type {@link TimestampedKeyValueStore}) will be backed by
```

Comment:
As above. Should we `TimestampedWindowStore` for this class.

Reply:
Updated